app/src/main/res/values/arrays.xml

```diff
@@ -64,6 +64,7 @@
         <item>@string/pvr</item>
         <item>@string/files</item>
         <item>@string/addons</item>
+        <item>@string/favourites</item>
```

Comment:
I think this should be in the same order as in the menu, so this entry should be after pvr, before files.